internal/fuse/dir.go

```diff
@@ -9,7 +9,7 @@ import (
 
 	"bazil.org/fuse"
 	"bazil.org/fuse/fs"
-	"golang.org/x/net/context"
+	"context"
```

Comment:
Shouldn't this now go in the first group of stdlib dependencies?

Reply:
I've moved the context imports to the stdlib block. Apparently the import ordering of gofmt works on a block basis, which generated the previous ordering.